### ISSUE TITLE
Skip to next and Skip to previous don't work

### ISSUE THREAD

**Issue by gabeseltzer** [dkanada/jellyfin-plugin-intros#58]
Hi! Thanks for an awesome plugin. I've found that when an intro gets added before a TV show episode, it messes up the relationships between episodes. This means that if you click the "Skip to Next" or "Skip to Previous" button, they won't work. I assume this is because, under the hood, the Intros plugin is manipulating the playlist and inserting the intro as the predecessor for the episode I'm currently watching. This seems to still linger even after disabling the plugin.

In short: if I start watching an episode of a show (say episode 2), the Intro plays, and then the episode plays. When I press "Skip to Previous," I expect episode 1 to start playing, but instead the Intro plays again. When I press "Skip to Next," I expect episode 3 to start playing, but instead nothing plays.

**Comment by gabeseltzer** [comment]
From reading some more, it looks like this is probably an issue with Jellyfin's implementation of the plugin interfaces! https://github.com/jellyfin/jellyfin-plugin-template#4a-implement-interfaces
